Committees/NPEC.adoc

```diff
@@ -0,0 +1,138 @@
+- Librarian
+- Teacher(s)
+
+=== Courses
+
+The specific structure and coursework is up to the discretion of the committee, but for each of these courses, the follow works **must** be incorporated:
+
+==== Capitalism 
+
+===== 101
+
+- Marx and Engels "Wage Labor and Capital"
+
+- Marx "Value, Price, and Profit"
+
+- Lenin "Imperialism: The Highest Stage of Capitalism"
+
+===== 201
+
+- Capital Vol I
+
+===== 225
+
+- Lenin "The Development of Capitalism in Russia"
+
+===== 301
+
+- Capital Vol II
+
+===== 401
+
+- Capital Vol III
+
+==== Scientific Socialism 
+
+===== 101
+
+- Stalin "Anarchism or Socialism?"
+
+- Lenin "The Three Sources and Three Component Parts of Marxism"
+
+- Lenin "State and Revolution"
+
+- Stalin "The Foundations of Leninism"
+
+- Mao "Oppose Book Worship"
+
+===== 201
+
+- Engels "Socialism: Utopian and Scientific"
+
+- Marx "Critique of the Gotha Program"
+
+- "Lenin on Socialist Democracy"
+
+===== 210
+
+- "A Short History of the Russian Revolution from 1905 to the present day (1937)"
+
+- Stalin "History of the CPSU-B: Short Course"
+
+===== 225
+
+- Mao "ANALYSIS OF THE CLASSES IN CHINESE SOCIETY"
+
+- Wright "On The Analysis of Classes in North American Society"
+
+- Foster "Toward Soviet America: Chapter 4"
+
+===== 240
+
+- "A Short History of the Russian Revolution"- Szamuely "First Models of the Socialist Economic Systems: Principles and Theories"
+
+- Stalin "Economic Problems of the USSR"
+
+===== 250
+
+- Mao "ON STATE CAPITALISM"
+
+- Mao "THE ONLY ROAD FOR THE TRANSFORMATION OF CAPITALIST INDUSTRY AND COMMERCE"
+
+===== 275
+
+- Mao "Concerning Economic Problems Of Socialism In The USSR"
+
+- Mao "Critique of Stalin’s Economic Problems Of Socialism In The USSR"
+
+===== 301
+
+- Mao "A Critique of Soviet Economics"
+
+===== 401
+
+- Liu Suinian, Wu Qungan "China's Socialist Economy: An Outline History"
+
+==== Party
+
+- "Lenin on Trade Unions"
+
+- Mao "Get Organized!"
+
+- Mao "On Practice"
+
+- Mao "Combat Liberalism"
+
+- Mao "On Contradiction"
+
+- Mao "On Correcting Mistaken Ideas in the Party"
+
+- Mao "Reform Our Study"
+
+- Mao "On Policy"
+
+- Lenin "Terms of Admission into Communist International"
+
+- Comintern "Guidelines on the Organizational Structure of Communist Parties, on the Methods and Content of their Work"
+
+==== Decolonization
+
+- Stalin "The National Question"
+
+- Lenin "The Right of Nations to Self-Determination"
+
+- Fanon "Wretched of the Earth"
+
+- Haywood "For A Revolutionary Position on the Negro Question"
+
+== Requirements
+
+Everyone on the committee **must** be able to prove comprehension of _any_ material they submit. Anyone considered for the committee must also be able to demonstrate an education in the works above either through PLT-Com or elsewhere. People should only teach what they are themselves profficient in. As well, this committee should work with lower level committees to provide assistance and guidance for their local educational efforts. 

```

Comment:
How frequently does this committee meet? How are we proving comprehension?

Reply:
They should meet at a minimum of 4 times a month. The CC should compile a test for each of the courses (tho, only when they are going to be run) and use that as a gauge.